### PR TITLE
Result

### PR DESCRIPTION
**Pull request by amantulsyan35**
Result page for the final score

### CODE REVIEW

File: Question/question.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.question-quiz-heading {
+  box-shadow: 0 2px 2px 2px rgba(0, 0, 0, 0.1);
+  padding: 1rem;
+  margin-bottom: 1rem;
+  display: flex;
+  justify-content: center;
+}
+
+.question-container {
+  display: flex;
```
Comment (parvez-rashid) [inline Question/question.css:17]: You can use a utility class for setting `display: flex`. I can see you have used the property in many places, setting a utility class would be easier.

File: Homepage/index.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  border: 1px solid rgb(0, 0, 0, 0.2);
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 20rem);
+  justify-content: center;
+  gap: 1rem;
+  padding: 1rem;
+}
+
+.quiz-category-card {
+  border: 1px solid black;
```
Comment (yaswanthmyneni) [inline Homepage/index.css:26]: changes: use css variables for colors

**Review by parvez-rashid — APPROVED** [review]
UI Review: Looks good, I feel there are too many borders for containers, it could've be a bit more subtle, just my perspective. The code looks great, well-structured and consistent. I do have a comment below, though not that important. Can be merged!

File: Result/result.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```
Comment (yaswanthmyneni) [inline Result/result.html:1]: UI changes: the 1st question on the results page is showing two right options.
Screenshot 2022-02-21 at 10 21 04 PM

**Review by yaswanthmyneni — APPROVED** [review]
You can merge it after changes.

File: Homepage/index.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+.homepage-container {
+  border: 1px solid rgb(0, 0, 0, 0.1);
```
Comment (pankaj-prog) [inline Homepage/index.css:2]: use color variables

File: Homepage/index.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  margin: 2rem 7rem 2rem 7rem;
+  padding: 1rem;
+  height: auto;
```
Comment (manojsarna) [inline Homepage/index.css:5]: you can use short hand , for eg : margin 2rem 7rem;

File: Homepage/index.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.homepage-quiz-heading {
+  box-shadow: 0 2px 2px 2px rgba(0, 0, 0, 0.1);
```
Comment (pankaj-prog) [inline Homepage/index.css:9]: color variable here as well

File: Homepage/index.html [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+      <div class="homepage-quiz-heading">
+        <h2>Anime Quiz</h2>
+      </div>
+      <div class="homepage-quiz-container">
+        <div class="quiz-category-card">x</div>
+        <div class="quiz-category-card">x</div>
+        <div class="quiz-category-card">x</div>
+        <div class="quiz-category-card">x</div>
+        <div class="quiz-category-card">x</div>
+      </div>
```
Comment (pankaj-prog) [inline Homepage/index.html:49]: use semantic tags

File: Homepage/index.css (hunk 2 above)
Comment (manojsarna) [inline Homepage/index.css:26]: Use Color Variables to ease the process, you can change the values directly from the root , you dont have to change it manually!!

File: Question/question.css [hunk 8]
```diff
@@ -0,0 +1,48 @@
+.quiz-question-container {
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  margin: 2rem 7rem 2rem 7rem;
+  padding: 1rem;
+  height: auto;
```
Comment (manojsarna) [inline Question/question.css:5]: check the margin values here also!!

File: Homepage/index.css [hunk 9]
```diff
@@ -0,0 +1,29 @@
+.homepage-container {
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  margin: 2rem 7rem 2rem 7rem;
```
Comment (pankaj-prog) [inline Homepage/index.css:3]: you can use the shorthand property ( 2rem, 7rem )

File: Question/question.css [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+.question-details {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.question-option-container {
+  border: 1px solid rgb(0, 0, 0, 0.2);
+  display: grid;
```
Comment (manojsarna) [inline Question/question.css:38]: use color variables

File: Question/question.css [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+  grid-template-rows: 1fr;
+  gap: 1rem;
+  padding: 1rem;
+}
+
+.quiz-option {
+  box-shadow: 0 2px 2px 2px rgba(0, 0, 0, 0.2);
```
Comment (manojsarna) [inline Question/question.css:45]: check for color vars

File: Result/result.css [hunk 12]
```diff
@@ -0,0 +1,54 @@
+@import url('https://enkidu-ui.netlify.app/css/style.css');
+
+.result-container {
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  margin: 2rem 7rem 2rem 7rem;
+  padding: 1rem;
```
Comment (manojsarna) [inline Result/result.css:6]: check for margin

File: Homepage/index.html [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+    <title>Anime-Quiz || Homepage</title>
+    <link rel="stylesheet" href="../base.css" />
+    <link rel="stylesheet" href="./index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
```
Comment (pankaj-prog) [inline Homepage/index.html:16]: you can import multiple css in css and then import a single css in html document

File: Result/result.css [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+  height: auto;
+}
+
+.result-heading {
+  border: 1px solid rgb(0, 0, 0, 0.2);
+  padding: 1rem;
```
Comment (manojsarna) [inline Result/result.css:12]: use color variables

File: Result/result.css [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+.result-details {
+  margin-bottom: 1rem;
+  box-shadow: 0 2px 2px 2px rgba(0, 0, 0, 0.1);
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+}
+
+.result-question-container {
+  border: 1px solid rgb(0, 0, 0, 0.2);
```
Comment (manojsarna) [inline Result/result.css:27]: check for  color variables

File: Result/result.css [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+  padding: 1rem;
+}
+
+.result-question {
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  padding: 1rem;
```
Comment (manojsarna) [inline Result/result.css:33]: check for color variables

File: Result/result.html [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+            <p>Kakuzu</p>
+          </div>
+          <div class="result-option">
+            <p>Indra</p>
+          </div>
+          <div class="result-right-option">
+            <p>Naruto</p>
+          </div>
+          <div class="result-option">
+            <p>Sakura</p>
```
Comment (pankaj-prog) [inline Result/result.html:76]: use list for options

File: Result/result.css [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+  gap: 1rem;
+  margin-bottom: 1rem;
+}
+
+.result-question p {
+  text-align: center;
+}
+
+.result-option {
+  border: 1px solid rgb(0, 0, 0, 0.2);
```
Comment (manojsarna) [inline Result/result.css:45]: check here also

**Review by pankaj-prog — CHANGES_REQUESTED** [review]

**Review by manojsarna — CHANGES_REQUESTED** [review]
overall its good!!

Do check the comments!!

**Review by eakansh125 — APPROVED** [review]
Some minor changes can be done (not mandatory for now):
1. A utility class for flex can be defined
2. A margin can be provided between different results for a better UI

Rest looks good from my side!

**Pull request merged**